src/libraries/System.Security.Cryptography/src/System/Security/Cryptography/RSACryptoServiceProvider.Windows.cs

```diff
@@ -322,7 +322,7 @@ public byte[] Encrypt(byte[] rgb, bool fOAEP)
 
             if (fOAEP)
             {
-                int rsaSize = (KeySize + 7) / 8;
+                int rsaSize = GetMaxOutputSize();
```

Comment:
In addition to these, there's a BitsToBytes helper (https://source.dot.net/#System.Security.Cryptography/System/Security/Cryptography/AsymmetricAlgorithmHelpers.Der.cs,842f18c74ce56b8c,references), of which some might be better self-documented as using the new method (unless ns2.0 interferes, or whatever).